src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -63,6 +64,10 @@ public function collect(Request $request, Response $response, \Exception $except
             if ('_route' === $key) {
                 $route = is_object($value) ? $value->getPath() : $value;
             }
+
+            if ('_route_params' === $key && is_array($value)) {
+                $routeParams = $value;
```

Comment:
Last commit fixes both request attributes as route attributes, in terms of looking great.

@javiereguiluz ill have a look.